VRDR.Messaging/BaseMessage.cs

```diff
@@ -102,7 +102,16 @@ protected void ExtractBusinessIdentifiers(DeathRecord from)
             {
                 this.CertificateNumber = certificateNumber;
             }
-            this.StateAuxiliaryIdentifier = from?.StateLocalIdentifier;
+            // take the first state local identifier if it exists
+            if (from?.StateLocalIdentifier != null && from.StateLocalIdentifier.Length > 0)
+            {
+                this.StateAuxiliaryIdentifier = from.StateLocalIdentifier[0].Item2;
```

Comment:
These shouldn't be an array.  They are separate variables.  Auxno can be empty when Auxno2 has a value.